demo/src/app/components/+dropdown/demos/disabled-item/disabled-item.html

```diff
@@ -0,0 +1,12 @@
+<div class="btn-group" dropdown>
+  <button dropdownToggle type="button" class="btn btn-primary dropdown-toggle">
+    Button dropdown <span class="caret"></span>
+  </button>
+  <ul *dropdownMenu class="dropdown-menu" role="menu">
+    <li class="disabled" role="menuitem"><a class="dropdown-item disabled" href="#">Action</a></li>
```

Comment:
class `disabled` will not disable link itself
so you should name this demo as `mark item as disabled`
not `make it disable item`

Reply:
fixed it